spec.html

```diff
@@ -32113,9 +32123,9 @@ <h1>FlattenIntoArray ( _target_, _source_, _sourceLen_, _start_, _depth_ [ , _ma
           <emu-alg>
             1. Assert: Type(_target_) is Object.
             1. Assert: Type(_source_) is Object.
-            1. Assert: _sourceLen_ is an integer Number &ge; 0.
-            1. Assert: _start_ is an integer Number &ge; 0.
-            1. Assert: _depth_ is an integer Number, *+&infin;*, or *-&infin;*.
+            1. Assert: ! IsNonNegativeInteger(_sourceLen_) is *true*.
+            1. Assert: ! IsNonNegativeInteger(_start_) is *true*.
+            1. Assert: ! IsNonNegativeInteger(_depth_) is *true*, or _depth_ is either *+&infin;* or *-&infin;*.
```

Comment:
No, `_depth_` could be a negative integer.

Reply:
```suggestion
            1. Assert: ! isInteger(_depth_) is *true*, or _depth_ is either *+&infin;* or *-&infin;*.
```

Reply:
(But with a capital "I": "IsInteger".)

Reply:
Fixed in a0061e5, thank you.